project-generator/src/main/resources/io/syndesis/project/converter/templates/pom.xml.mustache

```diff
@@ -16,6 +16,7 @@
     <camel.version>@camel.version@</camel.version>
     <funktion.version>@funktion.version@</funktion.version>
     <atlasmap.version>@atlasmap.version@</atlasmap.version>
+    <syndesis-connectors.version>@syndesis-connectors.version@</syndesis-connectors.version>
```

Comment:
ah sorry, misread the test. Its about the file content. But you could remove the property from the 'test-pull-push-pom.xml' fixture, too ?